proto/centauri/transfermiddleware/v1beta1/parachain_token_info.proto

```diff
@@ -5,6 +5,7 @@ import "gogoproto/gogo.proto";
 import "google/protobuf/timestamp.proto";
 
 option go_package = "x/transfermiddleware/types";
+option (gogoproto.goproto_getters_all) = false;
```

Comment:
why all getters was set to false? upstream changes?

Reply:
no getters or setters were being used for those 2 structures `ParachainIBCTokenInfo` `RemoveParachainIBCTokenInfo`